_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -349,6 +356,27 @@ public static function register_endpoints() {
 		) );
 	}
 
+	public static function get_plans( $request ) {
+		$data      = get_transient( 'jetpack_plans' );
+		$use_cache = apply_filters( 'cache_jetpack_plans', '__return_true', 1 );
```

Comment:
Could you add a docblock here? 

I'm also not quite sure `__return_true` is the best option here; it is usually meant to be used when adding filters. Just `true` may be easier to understand imo.

Reply:
🤦‍♂️